src/header/common/mod.rs

```diff
@@ -22,6 +22,7 @@ pub use self::allow::Allow;
 pub use self::authorization::{Authorization, Scheme, Basic, Bearer};
 pub use self::cache_control::{CacheControl, CacheDirective};
 pub use self::connection::{Connection, ConnectionOption};
+pub use self::content_disposition::ContentDisposition;
```

Comment:
This should also export `DispositionType` and `DispositionParam`, or else no can do much with this header.